apisix/consumer.lua

```diff
@@ -186,24 +204,43 @@ function _M.consumers()
 end
 
 
-local function create_consume_cache(consumers_conf, key_attr)
+local create_consume_cache
+do
+    local consumers_plugin_key_lrucache_tab = {}
+
+local function create_new_consumer(consumer)
```

Comment:
we need a better name, it is not creating a new consumer

it should be `transform` or `fill secret`, what do you think?